index.html

```diff
@@ -360,7 +451,7 @@ <h2>JSON Web Token Header Parameters</h2>
 </section>
 <section class="normative">
   <h2>Securing Verifiable Credentials</h2>
-  <p>The [[VC-DATA-MODEL]] describes the approach taken by JSON Web Tokens to securing claimsets as applying an <code>external proof</code>.</p>
+  <p>The <a data-cite="VC-DATA-MODEL#proof-formats"></a> describes the approach taken by JSON Web Tokens to securing claimsets as applying an <code>external proof</code>.</p>
```

Comment:
```suggestion
  <p>The <a data-cite="VC-DATA-MODEL#proof-formats"></a> describes the approach taken by JSON Web Tokens to secure claimsets as <i>applying an <code>external proof</code></i>.</p>
```
Or something, to help delineate that clause, to make this rather long and complex sentence easier to parse.